style.css

```diff
@@ -4,121 +4,249 @@ body {
+.introduction .introduction-title {
+  margin-left: 24px;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
-  display: flex;
-  align-items: center;
+  margin-top: 128px;
   color: #172b4d;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
 }
 
-.content {
-  width: 327px;
-  height: 146px;
-  top: 276px;
-  margin: 12px 24px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
+.introduction .introduction-desc {
   font-weight: 400;
   font-size: 16px;
   line-height: 24px;
+  margin-left: 24px;
+  width: 327px;
+  height: 146px;
   color: #344563;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
 }
 
-.social-links {
-  width: 117px;
-  height: 24px;
-  margin: 72px 234px 15.03px 24px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
+.introduction .connect-title {
+  margin-left: 24px;
+  margin-top: 12px;
   font-weight: 500;
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
 }
 
-.connect {
-  width: 170.17px;
-  height: 20px;
-  margin: 15.03px 180.83px 136px 24px;
-  top: 470px;
-  flex: none;
-  order: 3;
-  flex-grow: 0;
+@media screen and (min-width: 768px) {
+  .connect {
+    display: flex;
+    flex-direction: row;
+    margin-left: 24px;
+  }
+}
+
+.introduction .connect li {
+  padding: 6px;
+  font-size: 22px;
+  margin-top: 12px;
+}
+
+@media screen and (min-width: 768px) {
+  .introduction {
+    margin-top: 73px 8px 76.54px 34.6px;
+    position: absolute;
+    width: 1397.4px;
+    height: 721.46px;
+    background-image: url("Header\ bg.jpg");
+    background-repeat: no-repeat;
+    background-size: auto;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .introduction .introduction-title {
+    margin-left: 260px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .introduction .introduction-desc {
+    margin: 12px 260px 12px 260px;
+    width: 920px;
+    height: 98px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .introduction .connect-title {
+    margin-top: 12px;
+    margin-left: 260px;
+    width: 117px;
+    height: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    flex: none;
+    order: 3;
+    flex-grow: 0;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .introduction .connect {
+    display: flex;
+    flex-direction: row;
+    margin-top: 14px;
+    margin-left: 260px;
+    flex: none;
+    order: 4;
+  }
+}
```

Comment:
- [ OPTIONAL] Hello @Kidd254 ,I can see that you made some changes based on the previous review that your hamburger is still visible on the desktop version, but there are some changes that you still need to work on. when we run the desktop version the mobile menu still appeared ,as required to this project when it comes to desktop version the hamburger menu should disappeared, please fix this issue so that you can move to the next step. :wink: 

- [ OPTIONAL]Remove the top and bottom arrows from your project and use full screen on the desktop version. :point_down: 
Your Design 
![fixthisvieux](https://user-images.githubusercontent.com/88894525/194598838-24a5b649-e7c2-4d71-88bf-85a42fe900c6.png)

Figma Design
![FIGMA](https://user-images.githubusercontent.com/88894525/194602422-cab2a142-a7e0-4e27-9d43-8553487cc39d.png)